cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -1270,8 +1270,7 @@ getTxWitnesses (ShelleyTx Shelley.Tx {
                      }) =
     map ShelleyBootstrapWitness (Set.elems bootWits)
  ++ map ShelleyKeyWitness       (Set.elems addrWits)
- -- TODO node cli team
- ++ map (ShelleyScriptWitness . error "Shelley.MultiSigScript") (Map.elems msigWits)
```

Comment:
Oh, I didn't realise the change was going to be so simple (one fewer constructor wrapping it) :slightly_smiling_face:.